SH_Script_Extract_Original_Paper_Before_Container-Start/README.md

```diff
@@ -0,0 +1,47 @@
+
+
+\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#
+
+## 	Bash Script to extract a certain file from an ERC    
+######   following the ERC specification of the o2r project
+\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\#\# 
+
+
+**_Syntax:_**
+
+`./recover_original_paper_from_image.sh $1 $2 [$3]`
+
+Parameters:
+
+*  **$1** : specifies the ID / unique name of an ERC as a String
+   * i.e.:  "NameofERC" / "id"
```

Comment:
This is a temporary work around; 
since the tool is not yet connected to the platform and file storage is still to be discussed, ERC ID / name is currently used for storing the extracted paper inside a uniquely named directory.